crates/rooch-types/src/transaction/ethereum.rs

```diff
@@ -50,7 +48,8 @@ impl EthereumTransactionData {
             access_list: None,
             max_priority_fee_per_gas: None,
             max_fee_per_gas: None,
-            chain_id: Some(U256::from(RoochChainID::DEV.chain_id().id())),
+            // TODO create ethereum chain id in genesis init to pass through transaction validator tests
```

Comment:
We do not need to define an ethereum chain id, we just use the RoochChainID.

Reply:
Resolved to RoochChainID DEV.